distribution/lib/Standard/Visualization/0.0.0-dev/src/Table/Visualization.enso

```diff
@@ -59,11 +59,27 @@ prepare_visualization y max_rows=1000 =
    Column Limit
 max_columns = 250
 
-## PRIVATE
+##PRIVATE
+leading_trailing_whitespace_count : Column -> Integer | Nothing
+leading_trailing_whitespace_count col =
+    find_whitespace col = 
+        filtered = col.to_vector.filter (c-> c.is_a Text && (c.first.is_whitespace || c.last.is_whitespace))
+        filtered.length
+    if (col.value_type == Value_Type.Mixed || col.value_type.is_text) then find_whitespace col else Nothing
+
+##PRIVATE
+contains_atleast_one_in_list : Text -> Vector -> Boolean
+contains_atleast_one_in_list str list =
+    does_contain_list = list.map (char-> str.contains char)
+    filtered = does_contain_list.filter (c-> c)
```

Comment:
The identity function can also be written as `(_)`, or `c->c` (no need for parentheses if there is no space in the lambda).